contribs/gmf/src/directives/layertree.js

```diff
@@ -290,6 +290,7 @@ gmf.LayertreeController.prototype.updateLayerDimensions_ = function(layer, node)
       var value = this.dimensions[key];
       if (value !== undefined) {
         dimensions[key] = value;
+        node.dimensions[key] = value;
       }
```

Comment:
`node.dimensions` are the default dimension values for a node; it should not be modified